server/protocols/jmap/src/main/java/org/apache/james/jmap/utils/JsoupHtmlTextExtractor.java

```diff
@@ -33,6 +33,10 @@
 public class JsoupHtmlTextExtractor implements HtmlTextExtractor {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(JsoupHtmlTextExtractor.class);
+    public static final String BR_TAG = "br";
+    public static final String UL_TAG = "ul";
+    public static final String LI_TAG = "li";
+    public static final String P_TAG = "p";
 
```

Comment:
> I want to make i clear I can not handle every single HTML label and that this is best effort.

That is A BIG issue maybe we should move to a more generic solution or at list add some test with some random tag to see if the output is good enought.